recipes/typer/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "typer" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "a8c49418f20c1d1b04a34424dded86cebf1f87042a8e448c6faa63cf38b9c0af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

Comment:
conda-forge is no longer building packages for py<3.6, so I think we can consider dropping the `skip` and seeing if we can make this `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

if you want to make this explicit, you can change `python` to `python >=3.6` under both `host` and `run`